packages/rke2-calico/generated-changes/patch/templates/tigera-operator/02-tigera-operator.yaml.patch

```diff
@@ -1,6 +1,18 @@
 --- charts-original/templates/tigera-operator/02-tigera-operator.yaml
 +++ charts/templates/tigera-operator/02-tigera-operator.yaml
-@@ -29,7 +29,7 @@
+@@ -23,13 +23,19 @@
+           operator: Exists
+         - effect: NoSchedule
+           operator: Exists
++        - key: "node-role.kubernetes.io/control-plane"
```

Comment:
This appears to already tolerate all NoExecute/NoSchedule taints, regardless of key?

Reply:
well yeah I thought of that too, take a look at this https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.5/rke/templates/coredns.go#L123-L128 this is for coredns in rke1 and this still doesnt tolerate the role taint, according to the docs:

```
There are two special cases:

An empty key with operator Exists matches all keys, values and effects which means this will tolerate everything.

An empty effect matches all effects with key key1.
```
But empty key with effect doesnt seem to cut it